CMakeLists.txt

```diff
@@ -8,12 +8,6 @@ if(POLICY CMP0054)
   cmake_policy(SET CMP0054 NEW)
 endif()
```

Comment:
Can't we remove all these `cmake_policy` definition?

https://cmake.org/cmake/help/v3.4/manual/cmake-policies.7.html
> The `cmake_minimum_required()` command does more than report an error if a too-old version of CMake is used to build a project. It also sets all policies introduced in that CMake version or earlier to `NEW` behavior.